compiler/qsc_project/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! This module handles the logic that constitutes the Q# project system.
+//! This includes locating a manifest file in the filesystem, loading and parsing
+//! the manifest, and determining which files are members of the project.
+
+mod error;
+#[cfg(feature = "fs")]
+mod fs;
+mod manifest;
+mod project;
+
+pub use error::Error;
+#[cfg(feature = "fs")]
+pub use fs::StdFs;
+pub use manifest::{Manifest, ManifestDescriptor, MANIFEST_FILE_NAME};
```

Comment:
Does `MANIFEST_FILE_NAME` need to be pub from the module or is it an internal detail to the crate? Just curious.